### ISSUE TITLE
[Feature Request] Shuffle Music By Artist

### ISSUE THREAD

**Issue by LoganJFisher** [jmshrv/finamp#341]
Right now you can shuffle music in a given album by an artist, in your list of all songs, and in a playlist. I'd like to request the ability to also shuffle songs of all albums of a given artist. This can already be accomplished by creating a playlist containing all of their songs, but that's not the most sensible way to go about this, and it would make more sense to simply be able to shuffle on the artist's page.

**Comment by Chaphasilor** [comment]
This should land at the latest once the new redesign (pinned issue) is done. There are no technical issues preventing this afaik, the interface is just missing a few buttons ^^

If you know some Flutter by any chance I'm sure this would be easy to add, otherwise it could be a while before @jmshrv gets to implementing this, so you'll have to be patient :)

**Comment by jmshrv** [comment]
Duplicate of #35
